LiskHQ/lisk-nano
# Suggest making an outgoing transaction - Closes #1019

### What was the problem?

There was no hint to users that they should make an outgoing transaction to save their publicKey on the blockchain.

### How did I fix it?
I implemented an infobox that suggests to users without outgoing transactions to make one.

### How to test it?
- Create a new account 
- Send some LSK to it from another account
- Check the message in the new account.
- Click the button to send the transactions.

### Review checklist
- [ ] The PR solves #1019
